webapp/src/components/testing/Testing.vue

```diff
@@ -144,19 +165,18 @@
       <div v-if="stepRef === 1">
```

Comment:
```suggestion
      <div 
        v-if="stepRef === 1"
        class="flex flex-col space-y-4 md:space-y-8"
      >
```
Follow same style as training. Right now there is no gap between the divs:
<img width="600" alt="Screenshot 2024-08-13 at 14 06 34" src="https://github.com/user-attachments/assets/a53fe5b2-da95-4f86-9ff6-cc101a9deba6">